### ISSUE TITLE
[C+ newdot payment][$500] Web - Task -  Emojis, emoji picker icon,  and edit icon are disapeared when hovering over notification from completed task

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#27546]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open the website
2. Click the + icon from LHN
3. Click assign task
4. create a task by sharing with someone
5. Complete the task
6. Hover over the completed task notification
7. Observe the tool-tips

## Expected Result:
There should be emojis, emoji picker icon, and edit icon when hovering over the notification

## Actual Result:
 Emojis, emoji picker icon, and edit icon disappear after a seconds

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.70.5
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/9046948e-8247-4801-bc1b-02455b7a7717

https://github.com/Expensify/App/assets/93399543/333d3898-8c63-40c6-a6ef-503712e8beb7

**Expensify/Expensify Issue URL:**
**Issue reported by:** @misgana96
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1694694710329109

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01bc63337bbb697954</li>
        <li>Upwork Job ID: 1702725083154939904</li>
        <li>Last Price Increase: 2023-10-04</li>
<li>Automatic offers: </li>
<ul>
<li>dukenv0307 | Contributor | 27173615</li>
<li>misgana96 | Reporter | 27173617</li>
</ul></li>
    </ul>
</details>

**Comment by kadiealexander** [comment]
@allroundexperts should this go on hold for https://github.com/Expensify/App/issues/24644?

**Comment by kadiealexander** [comment]
This is no longer on hold and still reproducible:

![image](https://github.com/Expensify/App/assets/59587260/9743d0d4-2316-40a5-aa73-6182ac767100)

**Comment by kadiealexander** [comment]
@allroundexperts any thoughts on the proposals we've already received?

**Comment by kadiealexander** [comment]
@allroundexperts any thoughts on the proposal above?

**Comment by lakchote** [comment]
@dukenv0307 could you please share a video with the same steps as in the original video in the issue?

In your video, the task is in `incomplete` state when you're hovering to see emojis whereas in the video it's in `complete`state when the bug is happening. Just to be sure, thanks

**Comment by lakchote** [comment]
Thank you @dukenv0307

**Comment by kadiealexander** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@lakchote] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@lakchote] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@lakchote] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@allroundexperts] Determine if we should create a regression test for this bug.
- [ ] [@allroundexperts] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@kadiealexander] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by kadiealexander** [comment]
🎯 ⚡️ Woah @allroundexperts / @dukenv0307, great job pushing this forwards! ⚡️

The pull request got merged within 3 working days of assignment, so this job is eligible for a 50% #urgency bonus 🎉

- when @dukenv0307 got assigned: Oct 13 6:55am
- when the PR got merged: Oct 16 10:31am

On to the next one 🚀

**Comment by kadiealexander** [comment]
Payouts due:

Issue Reporter: $50 @misgana96
Contributor: $500 @dukenv0307
Contributor+: $500 @allroundexperts

Eligible for 50% #urgency bonus? [Yes](https://github.com/Expensify/App/issues/27546#issuecomment-1780497933)

Upwork job is [here](https://www.upwork.com/jobs/~01bc63337bbb697954).

**Comment by lakchote** [comment]
@allroundexperts can you please identify the PR that caused the bug? Thanks!

**Comment by allroundexperts** [comment]
## Checklist
1. There isn't any PR on the frontend which caused this. This was supposed to be fixed from the backend but due to time constraints, we decided to fix this on the frontend.
2. N/A
3. N/A
4. A regression test would be an overkill IMO given that this is really small. However, if there's a disagreement, then following are the steps.

## Regression test

1. Create a task -> Assign it to someone -> Complete the task
2. Hover over the completed task notification

Verify that the options inside the mini context menu do not change with time.

Do we 👍 or 👎 ?

**Comment by kadiealexander** [comment]
Closing as all tasks completed, @allroundexperts please request your payment via Newdot.

**Comment by JmillsExpensify** [comment]
$750 payment to @allroundexperts based on summary above.
